datumaro/util/image.py

```diff
@@ -65,7 +65,10 @@ def save_image(path, image, create_dir=False, dtype=np.uint8, **kwargs):
     if not kwargs:
         kwargs = {}
 
-    if _IMAGE_BACKEND == _IMAGE_BACKENDS.cv2:
+    backend = _IMAGE_BACKEND
```

Comment:
simple python example (check uint16 and int32):
```
>>> import cv2
>>> import numpy as np
>>> image = np.ones((1,5), dtype=np.uint8)*1001
>>> image
array([[1001, 1001, 1001, 1001, 1001]], dtype=uint16)
>>> cv2.imwrite("test.png", image)
True
>>> image_read = cv2.imread("test.png", cv2.IMREAD_UNCHANGED)
>>> image_read
array([[1001, 1001, 1001, 1001, 1001]], dtype=uint16)
>>> 
>>> 
>>> image = np.ones((1,5), dtype=np.int32)*1001
>>> image
array([[1001, 1001, 1001, 1001, 1001]], dtype=int32)
>>> cv2.imwrite("test.png", image)
True
>>> image_read = cv2.imread("test.png", cv2.IMREAD_UNCHANGED)
>>> image_read
array([[255, 255, 255, 255, 255]], dtype=uint8)
```

Reply:
Leave a comment in the code.

Reply:
Added